protocol/x/clob/ante/clob.go

```diff
@@ -60,6 +60,14 @@ func (cd ClobDecorator) AnteHandle(
 		return next(ctx, tx, simulate)
 	}
 
+	// Disable order placement and cancelation processing if the clob keeper is not hydrated.
+	if !cd.clobKeeper.IsHydrated() {
```

Comment:
Nit: would it makes sense to name this field more generically? Hydration is one of the things that could be done here
something like `IsInitialized`? This pattern can be consistent across modules